stl/src/atomic.cpp

```diff
@@ -14,23 +14,23 @@ _EXTERN_C
 volatile long _Shared_ptr_flag;

```

Comment:
Can you just use real `<atomic>` in here?

Reply:
I don't see a reason why not. But somehow it was not used in the first place, so I suspected some reason behind it.
Maybe not changing lock type. But it is in a library, so no subject to breaking anything.

Reply:
When this code was written `<atomic>` was a huge mess that would go through ~3 API calls minimum per operation; it's much better now :)